<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VMMMonitorDialog</class>
 <widget class="QMainWindow" name="VMMMonitorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>731</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tabCPUMem">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QLabel" name="labelCPU">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>-10</y>
        <width>711</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU Usage: VMM vs Whole Virtualizer</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelMem">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>711</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Memory Usage: VMM vs Whole Virtualizer</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCustomPlot" name="qcpCPU" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>711</width>
        <height>241</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="qcpMem" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>711</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabIO">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QCustomPlot" name="qcpIO" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>721</width>
        <height>471</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelIO">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>711</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Disk/Net I/O Dynamics in VMM</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabVMs">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTableWidget" name="tableVMs">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>731</width>
        <height>471</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="cursor" stdset="0">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>1</number>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>40</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>20</number>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <column/>
      <column/>
      <column/>
     </widget>
     <widget class="QLabel" name="labelVMs">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>-10</y>
        <width>711</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Per-VM Performance Statistics</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>734</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>Main Controls...</string>
    </property>
    <addaction name="actionPause"/>
    <addaction name="actionRestart"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMain"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
